src/pmsearch/pmsearch_daily.sh

```
@@ -0,0 +1,234 @@
+# The nightly update copies it, then adds/updates runtime data.
+BASE="$PCP_SHARE_DIR/lib/pcp.search"
+if [ -f "$BASE" ]
+then
+    cp "$BASE" "$INDEX"
+    if $VERBOSE
+    then
+	echo "$prog: copied base index from $BASE"
+    fi
+fi
+
+nentries=`grep -c '^@ ' $tmp/helptext`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)